### ISSUE TITLE
Sword URL Consistency

### ISSUE THREAD

**Issue by terrywbrady** [CDLUC3/mrt-doc#1065]
Reported by Scott:

Hi everyone.  Question about [sword-aws.cdlib.org](http://sword-aws.cdlib.org/) .  We see some datasets getting assigned a SWORD update URL with this domain intermittently and it doesn't seem to resolve.

[1:30](https://cdlib.slack.com/archives/C86U0VBLH/p1653510638567929)
when we change it to [mrtsword.cdlib.org](http://mrtsword.cdlib.org/) it seems to work for sending another version later.  I assume it's fine for us to change it this when when this happens? (edited) 
[1:32](https://cdlib.slack.com/archives/C86U0VBLH/p1653510734808829)
I'm also not sure how this crops up.  It looks like there were about 7 in the past couple weeks and Ryan said he saw a similar number a week or two ago.

**Comment by elopatin-uc3** [comment]
See also #419

**Comment by terrywbrady** [comment]
See #419

**Comment by elopatin-uc3** [comment]
Mark changed URL in sword-info.txt; redeploy required to make this permanent.

Mark also changed the URL on the production host, which will take effect on restart after patching.

**Comment by mreyescdl** [comment]
Modified Sword config and pushed to GitHub 
   ```https://github.com/CDLUC3/mrt-tomcat/commit/28ea01d0c3e8ff580f1f66940bafb0c3a8c1b4bf```

Also, copied this change to active Sword config on host mrtsword.cdlib.org so a redeployment is NOT necessary for it to take effect

**Comment by elopatin-uc3** [comment]
From Slack:

SWORD couldn’t make submissions asynchronously, post-patching.

Here’s what we discovered:

- The ingestUrlAdd and ingestUrlUpdate URLs in sword-info.txt were no longer usable by SWORD. Although we did not change these, pre-patching, SWORD could no longer reach them via merritt.cdlib.org:33121
- To work around this problem, we changed both ingestUrlAdd and ingestUrlUpdate to use a specific ingest server address (in this case ingest02).
- This sword-info.txt change was made locally on the SWORD production server. Puppet does not touch this file, so it will stay the same.

We suspect that perhaps there was a cname change associated with [merritt.cdlib.org](http://merritt.cdlib.org/) that didn’t manifest itself until patching reboots and will ask IAS tomorrow.
- Ashley followed up. The last cname change occurred a year ago, when the ALB was deployed.

Next steps:
- Determine a final fix for sword-info.txt
- Check this fix into [mrt-tomcat](https://github.com/CDLUC3/mrt-tomcat/blob/28ea01d0c3e8ff580f1f66940bafb0c3a8c1b4bf/data/mrtHomes/mrt-sword01x2-prd/sword-info.txt) so it will continue to be used on the next redeploy.

Proposed change to sword-info.txt:
ingestUrlAdd and ingestUrlUpdate could both use merritt-aws as a domain name.
